src/components/PadSettings.spec.js

```diff
@@ -215,7 +212,9 @@ describe('PadSettings', () => {
         />
       </MemoryRouter>
     );
-    userEvent.click(screen.getByRole('button', {name: 'play preview sample play preview'}))
-    expect(samplePreview).toHaveBeenCalled()
+    userEvent.click(
+      screen.getByRole('button', { name: 'play preview sample play preview' })
```

Comment:
Is there one "play preview" too much?

Reply:
No, I don't think so. The name was suggested to me in the terminal 😅